reactive-streams/src/main/scala/fs2/interop/reactivestreams/StreamSubscription.scala

```diff
@@ -87,6 +87,7 @@ private[reactivestreams] final class StreamSubscription[F[_], A] private (
         case Outcome.Canceled()  => cancelMe
       }
       .void
+      .handleError(_ => ()) // errors handled above and passed to subscriber
```

Comment:
```suggestion
      .voidError // errors handled above and passed to subscriber
```

Reply:
Oh, I initially tried replacing `.void` by `.voidError`, thinking that `.voidError` was an extra layer on top of `.void`. You made me realise it's not the case, thank you.